.gitignore

```diff
@@ -5,3 +5,5 @@ __pycache__/
 #file
 *.pyc
 .coverage
+
+qa_web/whoosh_index/
```

Comment:
This is a relative path and therefore currently ignores nothing. I believe it should rather be `SA3/qa_web/whoosh_index/` or `*/whoosh_index/`

Reply:
What a fool... Indeed. Thank you.